packages/api-specification/interface/window.ts

```diff
@@ -98,6 +98,10 @@ declare namespace ssf {
      * Makes the window transparent. Default is false.
      */
     transparent?: boolean;
+    /**
+     * The id of the display to show the window on.
```

Comment:
One quick question - I presume that if the `display` property is not present, the `x` and `y` properties are relative to the origin of the given display? Probably worth mentioning in the docs.